User flow language customization documentation (#10918)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/userflowlanguageconfiguration.md

---
title: "userFlowLanguageConfiguration resource type"

> description: "User flows language customization is a feature that allows a given user flow to support customization of multiple languages, from all the built-in languages to a custom language."

Reviewer: ```suggestion
description: "The userFlowsLanguageConfiguration object allows a user flow to support customization of multiple languages."
```
